### PR TITLE
Trigger coverage build on the master branch

### PR DESCRIPTION
**Pull request by bmagyar**

### CODE REVIEW

File: .github/workflows/ci-coverage-build.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name: Coverage Build
 on:
   workflow_dispatch:
-    branches:
-      - master
+    push:
+      branches:
+        - master
```
Comment (fmauch) [inline .github/workflows/ci-coverage-build.yml:6]: Shouldn't the dispatch and push trigger be two separate entries?
```suggestion
    branches:
      - master
  push:
    branches:
      - master
```

**Comment by christophfroehlich** [comment]
> ## [Codecov](https://app.codecov.io/gh/ros-controls/ros2_control/pull/1119?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=ros-controls) Report
> > Merging [#1119](https://app.codecov.io/gh/ros-controls/ros2_control/pull/1119?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=ros-controls) ([349cb61](https://github.com/ros-controls/ros2_control/commit/349cb613f7705694d7f997a436c611f2924ea32c)) into [master](https://app.codecov.io/gh/ros-controls/ros2_control/commit/a6a86d44c202652f8bda70b8b0ec385a96b14e81?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=ros-controls) ([a6a86d4](https://github.com/ros-controls/ros2_control/commit/a6a86d44c202652f8bda70b8b0ec385a96b14e81)) will **decrease** coverage by `0.01%`.
> > The diff coverage is `n/a`.
> 
> Additional details and impacted files

this looks fine now 👍

File: .github/workflows/ci-coverage-build.yml (hunk 1 above)
Comment (aprotyas) [reply .github/workflows/ci-coverage-build.yml:6]: I think this is fine..

**Review by aprotyas — APPROVED** [review]

**Review by destogl — APPROVED** [review]

**Comment by destogl** [comment]
thanks @fmauch

**Review by saikishor — APPROVED** [review]
LGTM

File: .github/workflows/ci-coverage-build.yml (hunk 1 above)
Comment (bmagyar) [reply .github/workflows/ci-coverage-build.yml]: we have it like tbis elsewhere too

**Pull request merged**